com.unity.multiplayer.mlapi/Tests/Editor/NetworkObjectTests.cs

```diff
@@ -0,0 +1,46 @@
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace MLAPI.EditorTests
+{
+    public class NetworkObjectTests
+    {
+        [Test]
+        public void GetBehaviourIndexNone()
+        {
+            var gameObject = new GameObject(nameof(NetworkObject));
+            var networkObject = gameObject.AddComponent<NetworkObject>();
+            
+            // TODO: Maybe not hardcode message?
+            LogAssert.Expect(LogType.Error, $"[MLAPI] Behaviour index was out of bounds. Did you mess up the order of your {nameof(NetworkBehaviour)}s?");
```

Comment:
Like you, I'm not crazy about this relying on the error message in NetworkObject.cs lines up exactly with this hard-coded string.   Since we already check for null below, is this just making sure we log an error message?

Reply:
This is actually not a check, but it's just saying "dont fail the test just because this error message comes through".

By default the test runner will fail if a error log is sent.

Reply:
does the test fail due to logs here without this trick currently?

Reply:
Yes

Reply:
@TwoTenPvP now that I know the truth, I feel no better :)